third_party/packages/cupertino_icons/pubspec.yaml

```diff
@@ -9,10 +9,12 @@ environment:
   sdk: ^3.3.0
 
 dev_dependencies:
+  collection: ^1.18.0
   flutter:
     sdk: flutter
   flutter_test:
     sdk: flutter
+  path: 1.9.0
```

Comment:
Why is this pinned to an exact version? `path` is allowed as an unpinned dependency in the repo.

Reply:
It shouldn't be `any`, it should be a standard `^` dependency.